jbpm-services/jbpm-kie-services/src/main/java/org/jbpm/kie/services/impl/RuntimeDataServiceImpl.java

```diff
@@ -761,6 +761,13 @@ public NodeInstanceDesc getNodeInstanceForWorkItem(Long workItemId) {
         return variablesState;
     }
 
+    @Override
+    public Collection<VariableDesc> getVariablesCurrentStateWithValueObject(Long processInstanceId) {
```

Comment:
It seems most of the logic around storying actual object was removed for more review, should this (and similar references in this PR to WithValueObject) still be here?

Reply:
removed. left behind by mistake.